cfpb/capital-framework
# Removed the deprecated font-icon system from cf-icons

Supersedes https://github.com/cfpb/capital-framework/pull/810 to target the master branch.
Also adds some 

## Removals

- Removes the deprecated font-icon system from cf-icons

## Changes

- Fixes space in button with icon spacing.

## Testing

1. Follow the [testing locally instructions](https://github.com/cfpb/capital-framework/blob/canary/CONTRIBUTING.md#testing-components-locally)
1. Run `gulp build` && `yarn start`
1. Navigate to `http://localhost:3000/components/cf-icons`
1. Confirm the icon font documentation is gone.
